target/linux/pistachio/config-5.15

```diff
@@ -187,8 +204,9 @@ CONFIG_NEED_DMA_MAP_STATE=y
 CONFIG_NET_FLOW_LIMIT=y
 CONFIG_NET_NS=y
 CONFIG_NET_PTP_CLASSIFY=y
+CONFIG_NET_SELFTESTS=y
 CONFIG_NLS=y
```

Comment:
I think we should remove these options:
````
CONFIG_NET_FLOW_LIMIT=y
CONFIG_NET_NS=y
CONFIG_NET_PTP_CLASSIFY=y
CONFIG_NET_SELFTESTS=y
CONFIG_NLS=y
```
in a separate patch .

Reply:
Why?

Reply:
These configurations are not target specific and should use the settings from the generic configuration. This was already a problem in the configuration for kernel 5.10.

Reply:
Okay, so doing a tree-wide patch for all targets moving those options to common.